man/scrot.txt

```diff
@@ -83,11 +83,16 @@ SELECTION STYLE
 
   The following specifiers are recognised:
 
-    style=(solid,dash),width=(range 1 to 8),color="value"
+    style=(solid,dash),width=(range 1 to 8),color="value",
+    opacity=(range 10 to 100),mode=(edge,classic)
 
-  The default style is:
```

Comment:
Check this, I don't remember seeing it changed

Reply:
"The default style is:"

https://github.com/resurrecting-open-source-projects/scrot/commit/6e9329beb69fcdb08ebb1c3ed8877892a9a6a6da#diff-969598d92e2975a61530e022fcc2aa590535d7d93ca94ab2ff7b1d9f4e879a2bR84